gluten-core/src/main/java/io/glutenproject/substrait/expression/ExpressionBuilder.java

```diff
@@ -225,7 +225,7 @@ public static void checkDecimalScale(int scale) {
   }
 
   public static ScalarFunctionNode makeScalarFunction(
-      Long functionId, ArrayList<ExpressionNode> expressionNodes, TypeNode typeNode) {
+      Long functionId, List<ExpressionNode> expressionNodes, TypeNode typeNode) {
```

Comment:
why need this change?

Reply:
It's a very unusual practice to define a variable with `ArrayList` type. Should use the abstract form as long as the code works.